packages/node_modules/@node-red/nodes/locales/en-US/network/21-httprequest.html

```diff
@@ -25,7 +25,7 @@ <h3>Inputs</h3>
         <dd>If not configured in the node, this optional property sets the HTTP method of the request.
             Must be one of <code>GET</code>, <code>PUT</code>, <code>POST</code>, <code>PATCH</code> or <code>DELETE</code>.</dd>
         <dt class="optional">headers <span class="property-type">object</span></dt>
-        <dd>Sets the HTTP headers of the request.</dd>
+        <dd>Sets the HTTP headers of the request. NOTE: Any headers set in the UI will overwrite any matching headers in <code>msg.headers</code> </dd>
```

Comment:
```suggestion
        <dd>Sets the HTTP headers of the request. NOTE: Any headers set in the node configuration will overwrite any matching headers in <code>msg.headers</code> </dd>
```

I think we tend to reference 'the node configuration' rather than the UI.